client/packages/common/src/utils/numbers/NumUtils.ts

```diff
@@ -29,6 +29,13 @@ export const NumUtils = {
     const multiplier = 10 ** dp;
     return Math.round(value * multiplier) / multiplier;
   },
+  /**
+   * Checks if precision is greater than 2. Some numbers have floating precision
+   * errors e.g. 2.05 * 100 = 204.99999999999997 so using 1000 for a more reliable result...
+   */
+  hasMoreThanTwoDp: (value: number): boolean => {
+    return ((value * 1000) / 10) % 1 !== 0;
+  },
```

Comment:
Nice! Javascript is weird.

I wonder is this new check 100% safe? or just safer? I guess just safer is fine if it is safe enough?

Reply:
Safer XD